CONTRIBUTING.md

```diff
@@ -646,192 +642,200 @@ If you have any problems with formatting, please ping the [formatting team](http
 
 ### How to help committers assess your PR
 
-For the committer to judge your intention, it's best to explain why you've made your change.
-This does not apply to trivial changes like version updates because the intention is obvious (though linking the changelog is appreciated).
-For any more nuanced changes or even major version upgrades, it helps if you explain the background behind your change a bit.
-E.g. if you're adding a package, explain what it is and why it should be in Nixpkgs.
+It's best to explain *why* you've made your change, because guessing the intention is not always possible.
+This does not apply to trivial changes like version updates, because the intention is obvious.
+For more nuanced changes or even major version upgrades, it helps if you explain the background behind your change.
+For example, if you're adding a package, explain what it is and why it should be in Nixpkgs.
 This goes hand in hand with [Writing good commit messages](#writing-good-commit-messages).
 
-For the code quality assessment, you cannot do anything yourself as only the committer can do this and they already have your code to look at.
-In order to minimise the need for back and forth though, do take a look over your code changes yourself and try to put yourself into the shoes of someone who didn't just write that code.
+To show the quality of your code, you should focus on making it *reviewable*.
+First, take a look at your code changes yourself and try to put yourself into the shoes of someone who didn't just write that code.
 Would you immediately know what the code does or why it is needed by glancing at it?
-If not, reviewers will notice this and will ask you to clarify the code by refactoring it and/or adding a few explanations in code comments.
-Doing this preemptively can save you and the committer a lot of time.
-To better convey the "story" of your change, consider dividing your change into multiple atomic commits.
+If not, reviewers will notice this and will ask you to clarify the code by refactoring it and/or adding code comments.
+Doing this preemptively can save a lot of time.
+Doing multiple unrelated changes in a single commit can become hard to review quickly.
+Thus, consider dividing your change into multiple atomic commits.
 There is a balance to strike however: over-fragmentation causes friction.
 
-The code artefacts are the hardest for committers to assess because PRs touch all sorts of components: applications, libraries, NixOS modules, editor plugins and many many other things.
-Any individual committer can only really assess components that they themselves know how to use however and yet they must still be convinced somehow.
-There isn't a good generic solution to this but there are some ways easing the committer's job here:
+The artifacts are the hardest to assess because PRs touch all sorts of components: applications, libraries, NixOS modules, editor plugins and many other things.
```

Comment:
Second time seeing "artifacts" used. Again, maybe this is a "me" issue, but I'm not clear what it means in this context.

Looking at the full file, "artifact" is present 3 times, none of which provide any definition or clarity. So I think we can treat this as jargon and either rephrase or define.

In this instance, it feels like it's referring back to the 3-point bullet list (despite that being in a different "section").